app/helpers/mod-if.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export function modIf([value, modBy]) {
+    return (value + 1 % modBy === 0);
```

Comment:
I think this will always return `false`, like `value + (1 % modBy) === 0`. Do we need this helper at all?